Add $\gamma_0 \gamma_i$ correlators in rho-pi-pi scattering

yannickdengler opened the pull request:
Compiled and ready for cluster :)

fzierler commented on Scattering/scatter.c:
```diff
@@ -185,6 +185,7 @@ int main(int argc, char *argv[])
     lprintf("MAIN", 0, "Configuration from %s\n", cnfg_filename);
     represent_gauge_field();
 
+            // YD: Going to 6 here to include g0gi
```
What is this comment referring to?

fzierler commented on LibHR/Observables/measure_scattering_tools.c:
```diff
@@ -901,15 +908,41 @@ void IOold_p(struct mo_p *molist[], int numsources, char *path, char *cnfg_filen
-				sprintf(tmp, "rho_p(%d,%d,%d)_g%d%d", px, py, pz, i + 1, j + 1);
-				lprintf("IOold_p", 0, "Printing for source %d momentum (%d,%d,%d) gamma %d\n", src, px, py, pz, i + 1, j + 1);
+				sprintf(tmp, "rho_p(%d,%d,%d)_g%d_g%d", px, py, pz, i + 1, j + 1);															// Slightly changed name to make it more understandable
+				// lprintf("IOold_p", 0, "Printing for source %d momentum (%d,%d,%d) gamma %d\n", src, px, py, pz, i + 1, j + 1);			// YD: Not needed
```
Can you remove the old code from the comments?

fzierler replied:
I fixed it for you.

fzierler commented:
It appears to be working correctly on the cluster where I tested it. We should write some documentation for the used functions in general.

fzierler commented:
I have looked at the $\gamma_0\gamma_i-\gamma_i$ cross-correlators in the free theory. The result is:
$C_{V-T}(\vec P, t-t')  = +\frac{4i}{L^6 T^2} \sum_{q_0, p_0, \mathbf p} e^{+i(p_0 - q_0) (t-t')} \frac{\left( \hat m(q_0,\vec P+\vec p) \sin(p_0) - \hat m(p) \sin(q_0) \right)}{N(p)N(q_0,\vec P+\vec p)}$
with 
$\hat m(p) =  m + \sum_\mu \left[ 1 -  \cos \left( p_\mu \right)\right]$
$N(p) = \hat m(p)^2 + \sum_\mu \sin^2 \left( p_\mu \right)$
I will do the same calculation for the triangle diagrams. We should then implement tests for the diagrams you have added.

yannickdengler commented:
Added a input option to measure axialvector.
Added the cross-correlator for gi to g0gi for P=0
Also added the contractions gi to gj for i!=j (Not sure if this one contains physics)
